pkg/options/types.go

```diff
@@ -106,56 +105,21 @@ func (r *ResourceSet) Type() string {
 }
 
 // NodeType represents a nodeName to query from.
-type NodeType map[string]struct{}
+type NodeType string
 
-// Set converts a comma-separated string of nodename into a slice and appends it to the NodeList
-func (n *NodeType) Set(value string) error {
```

Comment:
Let's keep the [`flag.Flag`](https://pkg.go.dev/flag#Value) behavior.

Reply:
It seems https://github.com/kubernetes/kube-state-metrics/pull/2388#discussion_r1655039540 was resolved without a fix? Can you PTAL?

Reply:
> Let's keep the [`flag.Flag`](https://pkg.go.dev/flag#Value) behavior.

Did you mean [`flag.Value`](https://pkg.go.dev/flag#Value)?

Reply:
Right, sorry, I meant that.